5.6/apache/Dockerfile

```diff
@@ -50,5 +50,5 @@ RUN { \
     && a2enmod rewrite
 
 # Install composer
-COPY install-composer /install-composer
+COPY image-files /
```

Comment:
> It's just one fewer ;), because you need to RUN something.

We already have `RUN` statements above where we can add this. Official docker images do the same. We should not add files just for the sake of it. Otherwhise you could move almost the whole install process into an external "setup" script and only COPY and RUN that. But that's not what we do: We try to keep simple commands inside the Dockerfile. And installing composer is a simple command. As I said before: Their example is unneccessarily long and boils down to 2 commands.